documentation/book/proc-using-openshift-builds-create-image.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-using-kafka-connect-with-plugins.adoc
+
+include::common/attributes.adoc[]
+
+[id='using-openshift-s2i-create-image-{context}']
+= Using {OpenShiftName} builds and S2I to create new images
+
+{OpenShiftName} supports link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^], which can be used together with the link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new container images.
```

Comment:
I think the links should be pointing to the downstream openshift docs if possible.

Reply:
Are they going to change the old releases documentation? I hope not because that would be quite stupid.

Reply:
Good question! Probably not, just the new releases would be different, but I don't know how they build the docs so the URLs might change.